setup.py

```diff
@@ -1,25 +1,180 @@
 # -*- coding: utf-8 -*-
 import os
+import re
 import io
+import json
+import hashlib
+import zipfile
+try:
+    # Python 2
+    from urllib2 import urlopen
+except ImportError:
+    # Python 3
+    from urllib.request import urlopen
+
 try:
     from setuptools import setup
     from setuptools.command.install import install
 except ImportError:
     from distutils.core import setup
     from distutils.core.command.install import install
+import distutils.cmd
+import distutils.log
 
-here = os.path.abspath(os.path.dirname(__file__))
+HERE = os.path.abspath(os.path.dirname(__file__))
 
-version_ns = {}
-with open(os.path.join(here, 'qtawesome', '_version.py')) as f:
-    exec(f.read(), {}, version_ns)
+VERSION_NS = {}
+with open(os.path.join(HERE, 'qtawesome', '_version.py')) as f:
+    exec(f.read(), {}, VERSION_NS)
 
-with io.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+with io.open(os.path.join(HERE, 'README.md'), encoding='utf-8') as f:
     LONG_DESCRIPTION = f.read()
 
+
+class UpdateFA5Command(distutils.cmd.Command):
```

Comment:
Please move this class to a file called `setupbase.py` next to `setup.py`. That way it'll be easier to read our `setup.py`.

Thanks!

Reply:
Done! (Sorry it took a little while to address this. :santa:)